api/config/config_prod.json

```diff
@@ -152,9 +152,12 @@
   },
   "reporting": {
     "dataset": "reporting_prod",
-    "maxRowsPerInsert": 500,
+    "maxRowsPerInsert": 10000,
     "batchSizeOverrides": {
-      "dataset_domain_value": 50000
+      "dataset_domain_value": 50000,
+      "leonardo_app_usage": 500,
```

Comment:
These 3 tables fail with "Request payload size exceeds the limit: 10485760 bytes" when the batch size is larger.